features/machine/machine.feature

```diff
@@ -68,3 +68,31 @@ Feature: Machine features testing
       | https://machine-api-operator.openshift-machine-api.svc:8443/metrics          | # @case_id OCP-25652
       | https://cluster-autoscaler-operator.openshift-machine-api.svc:9192/metrics   | # @case_id OCP-26111
       | https://machine-approver.openshift-cluster-machine-approver.svc:9192/metrics | # @case_id OCP-26102
+
+
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-27627
+
+ @admin
+ Scenario: Verify all machine instance-state should be  consistent with their providerStats.instanceState
+   Given I have an IPI deployment
+   When I run the :get admin command with:
+      | resource  | machines                           |
+      | namespace | openshift-machine-api |
+      | o         | jsonpath='{range .items[*]}{.metadata.annotations}{.status.providerStatus.instanceState}{"/\\n"}{end}'|
```

Comment:
The 3-line tables need to be aligned, i.e. the tailing `|` of each line should be aligned. There are some tools to auto-align such tables. Please also refer to https://github.com/openshift/verification-tests/blob/master/doc/vim.adoc

Reply:
done